scripts/run_cucumber_tests.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+set -e
+
+# we do not run `osrm-routed-js` tests on v12 since fastify doesn't support it
+NODE_VERSION=$(node --version)
+NODE_MAJOR_VERSION=$(echo $NODE_VERSION | cut -d. -f1)
+if [[ $NODE_MAJOR_VERSION != "v12" ]]; then
```

Comment:
I decided to not run `mmap` tests here until we won't fully migrate to JS-based osrm-routed to save CI time.